pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["jupyter_packaging~=0.7.9", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+requires = ["jupyter_packaging~=0.7", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"]
```

Comment:
This makes the dependency LESS restrictive, right?  allowing a regression to < 0.7.9?

Reply:
hm, good catch.  I want >= for all of those really. Will update to that.